appveyor.yml

```diff
@@ -56,12 +56,13 @@ install:
         Where-Object pullRequestId -eq $env:APPVEYOR_PULL_REQUEST_NUMBER)[0].buildNumber) { `
           throw "There are newer queued builds for this pull request, failing early." }
 
-    # We don't need to build the example recipe.
-    - cmd: if exist recipes\example rmdir recipes\example /s /q
-    # Skip if we are not in a PR.
-    - cmd: if "%APPVEYOR_REPO_NAME%" == "conda-forge/staged-recipes" if not defined APPVEYOR_PULL_REQUEST_NUMBER rmdir /q /s recipes
-    # Make sure the `recipes` directory exists if we removed it.
-    - cmd: if not exist recipes mkdir recipes
+    # Find the recipes from master in this PR and remove them.
+    - cmd: echo Finding recipes merged in master and removing them.
+    - cmd: cd recipes
+    - cmd: git fetch https://github.com/conda-forge/staged-recipes.git master
+    - cmd: |
+             for /f "tokens=*" %%a in ('git ls-tree --name-only FETCH_HEAD -- .') do rmdir /s /q %%a && echo Removing recipe: %%a
```

Comment:
Had to break onto a separate line like this as the yaml parser was getting confused by the `:` and thinking it was part of a dictionary. Doing this fixed it.